src/types/index.ts

```diff
@@ -26,11 +27,12 @@ export type Language = "fi" | "en-gb";
 export interface DataGridRow {
```

Comment:
This seems to be only used in the `VacationRequestsTable` so name this more specifically.